scripts/pay-id.v1.yml

```diff
@@ -0,0 +1,38 @@
+info:
+  title: test
+  version: '1.0'
+servers:
+  - url: 'http://localhost:3000'
```

Comment:
This seems a little odd.

For public GET APIs, the "payment pointer server" is whatever the payment pointer is. So `$example.com/hansbergren` would just be a GET request to `https://example.com/hansbergren`.

The Private API has a server, but that's not described here.

Reply:
I deleted this file. If you look at the right file, you'll see server isn't defined since there is no set endpoint.